extensions/kube-object-event-status/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kube-object-event-status",
-  "version": "0.0.1",
+  "version": "5.5.0-alpha.0.1649256751918",
```

Comment:
Please remove these changes to package.json files.

Reply:
Oops, I thought I removed these...